internal/flags/flags.go

```diff
@@ -115,4 +115,7 @@ const (
 	Normalization                   = "normalization"                   // Normalization flag
 	Backwards                       = "backwards"                       // Backwards flag
 	Strength                        = "strength"                        // Strength flag
+	LogTypes                        = "logTypes"                        // LogTypes flag
```

Comment:
there's already a `Type` const in this file, let's use that one.
we don't need the log part in the flag since are already in a log command